### PR TITLE
docs: update main README.md

### PR DESCRIPTION
**Pull request by bbogdanov**
Updating the main README.md to incoming v5 changes.

Signed-off-by: Bogdan Bogdanov <bbogdanov@vmware.com>

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,9 @@
...
 # Clarity Design System
 
-[![Build Status](https://travis-ci.com/vmware/clarity.svg?branch=master)](https://travis-ci.com/vmware/clarity)
-
-[![Build Status](https://saucelabs.com/browser-matrix/claritydesignsystem.svg)](https://saucelabs.com/beta/builds/b16110e384ce459ab68f10da6e38a285)
-
-[![npm version](https://badge.fury.io/js/%40clr%2Fcore.svg)](https://badge.fury.io/js/%40clr%2Fcore)
+| Build                                                                                                           | NPM                                                                                                  |
+| --------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [![Build Status](https://travis-ci.com/vmware/clarity.svg?branch=master)](https://travis-ci.com/vmware/clarity) | [![npm version](https://badge.fury.io/js/%40clr%2Fcore.svg)](https://badge.fury.io/js/%40clr%2Fcore) |
```
Comment (bdryanovski) [inline README.md]: I think that these build/package icons are not valid anymore.

* We don't use Travis CI for a long time so we need to update them with the Github Action more [here](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge)
* NPM Version badge is targeting `clr/core` package that is not the only package that we have. So adding other packages will be a best


npm version core
npm version angular
npm version cds angular - when released
npm version ui
npm version icons

```
![npm version icons](https://img.shields.io/npm/v/@clr/icons/latest?label=%40clr%2Ficons&style=flat-square)
```

**Review by bdryanovski — CHANGES_REQUESTED** [review]
Small changes related to the badges and build status

**Review by bdryanovski — APPROVED** [review]

**Review by hippee-lee — APPROVED** [review]

**Comment by bbogdanov** [comment]
@gnomeontherun let me know when I can merge this one

**Comment by mathisscott** [comment]
@bbogdanov : Is this going to be merged?

**Comment by bbogdanov** [comment]
Waiting @gnomeontherun for confirmation cause he had some improvement suggestions last time we spoke about this PR

**Comment by gnomeontherun** [comment]
I don't remember what I was going to say, so 👍

**Pull request merged**